spec/features/projects/project_custom_fields/overview_page/dialog/validation_spec.rb

```diff
@@ -290,6 +290,12 @@
           let(:custom_field) { integer_project_custom_field }
 
           it_behaves_like "a calculated custom field input"
+
+          it "displays the custom field label without the required asterisk" do
+            expect(page).to have_css("span", text: calculated_from_int_project_custom_field.name) do |label|
+              expect(label).to have_no_css("span", text: "*")
+            end
+          end
```

Comment:
Can be part of shared  examples if a variable is created for `calculated_from_int_project_custom_field`